howso/ablation.amlg

```diff
@@ -111,23 +111,21 @@
 
 	;reduce the trained data by removing cases which have an influence weight entropy that falls above
 	; a threshold.
-	;
-	;parameters:
-	; distribute_weight_feature: optional, default !autoAblationWeightFeature. name of feature whose values to use as case weights
-	; features: default all trained features. a list of features to use when computing influence weight entropies
-	; influence_weight_entropy_threshold: optional, default !autoAblationInfluenceWeightEntropyThreshold. cases with
-	; 	influence weight entropy above this quantile will be removed
-	; skip_auto_analyze: default false. whether to skip auto analyzing as cases are removed
 	#reduce_data
 	(declare
 		(assoc
 			;{type "string"}
+			;optional, default !autoAblationWeightFeature. name of feature whose values to use as case weights
```

Comment:
Lets remove the 'optional' from the text descriptions. In this case its actually not optional, but even in the places where the default value is (null) we will have that automatically inspected (or at least plan to).

Reply:
Should be resolved, but will allow you to review and resolve comment.